scripts/nvbench_compare.py

```diff
@@ -269,6 +300,19 @@ def extract_value(summary):
                     failure_count += 1
                     status = Fore.RED + "SLOW" + Fore.RESET
 
+                if has_batch_data:
+                    if (
+                        abs(frac_diff_batch) <= 0.01
+                    ):  # TODO(bgruber): what value to use here?
```

Comment:
I have no idea, let's get some input internally on that.

Reply:
Maybe just pick a sensible default and let the user override with command-line opts?